src/uct/api/uct.h

```diff
@@ -3037,6 +3069,8 @@ ucs_status_t uct_listener_create(uct_cm_h cm, const struct sockaddr *saddr,
  */
 void uct_listener_destroy(uct_listener_h listener);
 
+ucs_status_t uct_listener_query(uct_listener_h listener,
```

Comment:
I see the new api but I don't see any documentation for it

Reply:
good catch

Reply:
@yosefe brief documentation is not enough. We need full documentation for every routine.

Reply:
"listener" field description should start upper case for consistency